nncf/torch/checkpoint_loading.py

```diff
@@ -337,13 +337,13 @@ def run(self) -> Dict[str, torch.Tensor]:
                 "Legacy NNCF-enabled .pth checkpoint has been loaded! "
                 "The version-agnostic `RELU` operator name entries in the state dict "
                 "have been deprecated. "
-                "The loader will try to match these entries to the correspoindig `relu` and `relu_` op "
+                "The loader will try to match these entries to the corresponding `relu` and `relu_` op "
                 "names. The newly exported checkpoints will be adjusted to the new format."
             )
 
         if normalized_keys_to_load.has_legacy_storage_keys:
-            from nncf.torch.nncf_network import CURRENT_EXTERNAL_QUANTIZERS_STORAGE_PREFIX
-            from nncf.torch.nncf_network import LEGACY_EXTERNAL_QUANTIZERS_STORAGE_PREFIX
+            from nncf.torch.quantization.external_quantizer import CURRENT_EXTERNAL_QUANTIZERS_STORAGE_PREFIX
+            from nncf.torch.quantization.external_quantizer import LEGACY_EXTERNAL_QUANTIZERS_STORAGE_PREFIX
```

Comment:
Remove the legacy prefix altogether, we've supported it long enough.

Reply:
It's out of scope of this PR and ticket

Reply:
You had to touch this to implement this PR, which makes this relevant, and since you had to touch this, you can now do better - make it right.

Reply:
Please create ticket for this task.

Reply:
121535